app/services/encryptor.rb

```diff
@@ -0,0 +1,15 @@
+class Encryptor
+  ENCRYPTOR = ActiveSupport::MessageEncryptor.new(Rails.application.secrets.secret_key_base[0..31])
```

Comment:
We should look into this more - Rails examples seem to do a lot more:

https://api.rubyonrails.org/classes/ActiveSupport/MessageEncryptor.html

I don't understand this yet!

Reply:
Looks like the only extra feature is specifying a purpose. I'm not sure if it's necessary? 🤔

Reply:
The keys that are generated by the encryptor are quite long though, but not sure if that's something we can customise due to the way that encryption works.